onnxruntime/contrib_ops/cpu/bert/group_query_attention.cc

```diff
@@ -142,10 +142,14 @@ Status GroupQueryAttention<T>::Compute(OpKernelContext* context) const {
     const T* k_input;
     T* q_rotary;
     T* k_rotary;
+    // New OrtValues so data doesn't get freed
+    OrtValue Q_value_og;
+    OrtValue K_value_og;
     if (packed_qkv) {
       OrtValue RotaryQKV;
```

Comment:
The root cause is `OrtValue RotaryQKV` is declared within if scope. The memory will be released out of the if scope. The solution is to move `OrtValue RotaryQKV` to the outer scope and make sure it lives until kernel execution is finished.

I think the file has many place with similar issue. Please double check each OrtValue.